### PR TITLE
75 - Fixed Conventions Documentation link

### PR DESCRIPTION
**Pull request by lastav5**
https://github.com/ravendb/bootcamp/issues/75

### CODE REVIEW

**Review by ayende — COMMENTED** [review]
That puts us at an (old) version.
You can use the latest here: 
https://ravendb.net/docs/article-page/5.1/csharp/client-api/configuration/conventions

**Comment by ayende** [comment]
And the actual latest link that works is: https://ravendb.net/docs/article-page/latest/csharp/client-api/configuration/conventions

**Comment by lastav5** [comment]
Done

File: src/unit1/lesson4/README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ A lot of thought and effort was put into ensuring you won't have to touch
 the conventions. But you can do it every time you need to.
 
 We will not touch RavenDB conventions right now simply because we don't need
-to do it. If you want to know more, you can access the [RavenDB conventions documentation](https://ravendb.net/docs/article-page/latest/csharp/client-api/configuration/conventions/what-are-conventions).
+to do it. If you want to know more, you can access the [RavenDB conventions documentation](https://ravendb.net/docs/article-page/lastest/csharp/client-api/configuration/conventions).
```
Comment (ayende) [inline src/unit1/lesson4/README.md]:
```suggestion
to do it. If you want to know more, you can access the [RavenDB conventions documentation](https://ravendb.net/docs/article-page/latest/csharp/client-api/configuration/conventions).
```

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/unit1/lesson4/README.md (hunk 1 above)
Comment (lastav5) [reply src/unit1/lesson4/README.md]: Done

**Review by ayende — APPROVED** [review]

**Pull request merged**